packages/p2p/src/components/online-status/online-status-label.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { Localize } from 'Components/i18next';
+import { observer } from 'mobx-react-lite';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+const LastSeenLabel = ({ is_online, last_online_time }) => {
+    if (!is_online) {
+        if (last_online_time) {
+            const current_date = new Date();
+            const last_online_date = new Date(last_online_time * 1000);
+            const diff = moment.duration(moment(current_date).diff(last_online_date));
+            const addPrural = duration => (duration !== 1 ? 's' : '');
+
+            if (diff.months())
+                return (
+                    <Localize
+                        i18n_default_text='Seen {{ duration }} month{{ prural }} ago'
+                        values={{ duration: diff.months(), prural: addPrural(diff.months()) }}
+                    />
+                );
+            if (diff.days())
+                return (
+                    <Localize
+                        i18n_default_text='Seen {{ duration }} day{{ prural }} ago'
+                        values={{ duration: diff.days(), prural: addPrural(diff.days()) }}
+                    />
+                );
+            if (diff.hours())
+                return (
+                    <Localize
+                        i18n_default_text='Seen {{ duration }} hour{{ prural }} ago'
+                        values={{ duration: diff.hours(), prural: addPrural(diff.hours()) }}
+                    />
+                );
+            if (diff.minutes())
+                return (
+                    <Localize
+                        i18n_default_text='Seen {{ duration }} minute{{ prural }} ago'
+                        values={{ duration: diff.minutes(), prural: addPrural(diff.minutes()) }}
+                    />
+                );
+        } else {
```

Comment:
```suggestion
          const activity =
                (diff.months() && { data: `${diff.months()} month` }) ||
                (diff.days() && { data: `${diff.days()} day` }) ||
                (diff.hours() && { data: `${diff.hours()} hour` }) ||
                (diff.minutes() && { data: `${diff.minutes()} minutes` });

                return (
                    <Localize
                        i18n_default_text='Seen {{ duration }}{{ prural }} ago'
                        values={{ duration: activity, prural: addPrural(activity) }}
                    />
        } else {
```
What do you say if instead of several if we'll use this or something like this?